typings/tests/global-variables.types.ts

```diff
@@ -0,0 +1,37 @@
+Feature() // $ExpectError
+Scenario() // $ExpectError
+Before() // $ExpectError
+BeforeSuite() // $ExpectError
+After() // $ExpectError
+AfterSuite() // $ExpectError
+
+Feature('feature') // $ExpectType FeatureConfig
+
+Scenario('scenario') // $ExpectType ScenarioConfig
+Scenario(
+  'scenario',
+  {}, // $ExpectType {}
+  () => {} // $ExpectType () => void
+)
+Scenario(
+  'scenario',
+  () => {} // $ExpectType () => void
+)
+const callback: CodeceptJS.HookCallback = () => {}
+Scenario(
+  'scenario',
+  callback // $ExpectType HookCallback
+)
+Scenario('scenario',
+  (args) => {
+    args // $ExpectType SupportObject
+    args.I // $ExpectType I
+  }
+)
+
+// Before((
```

Comment:
@Vorobeyko please remove commented code

Reply:
Its draft PR) I'm still working on it

Reply:
@Vorobeyko ok sorry, I just saw it now :/